tests/sequence_sharding_optimistic/data/db1.increment.sql

```diff
@@ -0,0 +1,33 @@
+use `sharding_seq_opt`;
+
+insert into t2 (id, c1, c2) values (200003, 'twelve', 'twelfth');
+
+-- at this point:
+--      t1(id, c1, c2)
+--      t2(id, c1, c2)
+
+alter table t1 drop column c1;
+insert into t1 (id, c2) values (100003, 'fifteenth');
+
+-- at this point:
+--      t1(id,     c2)
+--      t2(id, c1, c2)
+
+alter table t2 add column c3 int;
+alter table t2 drop column c1;
```

Comment:
How about inserting some DMLs between these DDLs?

Reply:
added more DMLs in https://github.com/pingcap/dm/pull/568/commits/26f9d298acefe391f0633402bd0a84e3a62b2000.